### ISSUE TITLE
Matching input states

### ISSUE THREAD

**Issue by knownasilya** [tradegecko/ember-semantic-test-helpers#25]
Something like:

- `let value = await valueFor('label name')`
- `let required = await requiredFor('label name')`
- ...

**Comment by billybonks** [comment]
currently i have `assert.input('label name').hasValue('awesome')` working on adding it atm

**Comment by knownasilya** [comment]
Yeah, I use qunit-dom and basically wanted to do it by labels instead of by data-test-* or placeholder, etc..

**Comment by selvagsz** [comment]
#26

**Comment by billybonks** [comment]
Will hopefully get this merged on the weekend

**Comment by knownasilya** [comment]
@billybonks did you get a chance to work on this some more?
